lms/static/js/spec/commerce/receipt_spec.js

```diff
@@ -0,0 +1,203 @@
+define([
+        'jquery', 'jquery.ajax-retry',
+        'js/commerce/views/receipt_view', 'common/js/spec_helpers/ajax_helpers'
+    ],
+    function ($, AjaxRetry, ReceiptView, AjaxHelpers){
```

Comment:
because of https://github.com/edx/edx-platform/pull/12401/files/74d697952d7004aa715df532def3ffdbc255bd37#diff-51cca05dc2d297d186cdc94054145f1eR153